### ISSUE TITLE
Over 1K searches and more than 50 OTp will be blocked for day.

### ISSUE THREAD

**Issue by saurabh1069** [bombardier-gif/covid-vaccine-booking#506]
Today TOI Front page news .use your searches and OTP wisely
![image](https://user-images.githubusercontent.com/26090183/121458976-06d1ff80-c9c8-11eb-9ced-80af0a6e9677.png)

**Comment by BaasilSiddiqui** [comment]
![image](https://user-images.githubusercontent.com/84833531/121468960-49e89e80-c9d9-11eb-85d1-13ae9effd2e2.png)
ig party's over

**Comment by salian** [comment]
This change just makes life annoying for regular web users who have to keep entering OTPs, keep track of rationing their search quota over the day, and cannot refresh fastest fingers first for fear of getting blocked.

It makes little difference for automated solutions which can tirelessly follow all restrictions to the letter.

It always pays to understand booking strategy rather than leave scripts running for long periods. Know slot opening times, look for late night cancellations, and don't insist on a particular center, be willing to travel a bit. I'll add a section on this to the wiki.

In the long run I don't think blocking on CoWin will stand up to legal scrutiny. Being that governments have monopolized vaccine distribution, banning someone from the only way to get lifesaving vaccines is tantamount to a violation of Art 21 the right to life. Naturally people will clamour for lifesaving vaccines and try their best to get it in a shortage. No one is hoarding vaccines or wasting them, this is just about getting a spot in the queue.

The sole distributor can say sorry come in queue, or even that stocks are over come again tomorrow, but it can't ban anyone permanently for sure.

**Comment by saurabh1069** [comment]
@salian i completely agree with you. This will only makes life difficult for users as of one examples.
![image](https://user-images.githubusercontent.com/26090183/121476379-35f66a00-c9e4-11eb-834a-aa4fe5f876fd.png)

**Comment by Nakul93** [comment]
Totally agree with you @salian. Whole point of different bots for booking/notifying existing in the first place was due to the incompetence of the portal not able to make the entire process smooth. 
As mentioned [here](https://github.com/bombardier-gif/covid-vaccine-booking/issues/475#issuecomment-855346194) cowin portal should have some sort of a reservation queue. I recently came to know that thats how the vaccination bookings are carried out in the UK, where one has to register and get in a queue and if he/she is eligible then they send an invite using which they can book a slot.
Cowin portal at the bare minimum should have slot opening alerts in their apps(like that telegram bots)

**Comment by mrms123** [comment]
@Nakul93 - They overengineered tbh.
Akamai had a ready to deploy COVID Vaccine booking system, which is what is used both in UK and parts of US - and integrates directly into Salesforce.

I understand localization etc. are concerns, but I really don't see why they had to build CoWIN from scratch

https://www.zdnet.com/article/akamai-aims-to-improve-covid-19-registrations-integrates-vaccine-edge-with-salesforces-vaccine-cloud/

>Vaccine Edge manages registration workloads by passing users to a waiting room when loads are too high. The user maintains their location in the queue. Vaccine Edge is built with denial-of-service protection.

**Comment by sudo-u** [comment]
> Totally agree with you @salian. Whole point of different bots for booking/notifying existing in the first place was due to the incompetence of the portal not able to make the entire process smooth.
> As mentioned [here](https://github.com/bombardier-gif/covid-vaccine-booking/issues/475#issuecomment-855346194) cowin portal should have some sort of a reservation queue. I recently came to know that thats how the vaccination bookings are carried out in the UK, where one has to register and get in a queue and if he/she is eligible then they send an invite using which they can book a slot.
> Cowin portal at the bare minimum should have slot opening alerts in their apps(like that telegram bots)

Yes..Cowin portal sucks ..it should have reservation queue and it should also automatically schedule 2nd dose appointment.

  happen to stumble upon twilio's automated vaccine appointment booking system article  [here](https://www.twilio.com/blog/automated-vaccine-appointment-system-twilio-studio)

**Comment by BaasilSiddiqui** [comment]
@salian ig they aren't doing the reservation system since they they don't hv enough vaccines to vaccinate everybody people will see a large no. in queue and realize how short we are on vaccines. I don't see any other reason for not deploying an already devoloped system other than hiding their incompetancy

**Comment by sudo-u** [comment]
now telegram bots under45.in & above45.in are also illegal ? 
https://twitter.com/BertyThomas/status/1402640421335945220
